api/src/swagger/config.yaml

```diff
@@ -48,4 +50,266 @@ paths:
         "200":
           description: "Successful operation"
         "404":
-          description: "Invalid input"
\ No newline at end of file
+          description: "Invalid input"
+  /compartments/{id}:
+    get:
+      tags:
+      - "compartment"
+      summary: "For a given compartment name (e.g., Golgi apparatus or
+        golgi_apparatus), returns all containing metabolites, genes, reactions
+        and subsystems."
+      description: "For a given compartment name (e.g., Golgi apparatus or
+        golgi_apparatus), returns all containing metabolites, genes, reactions
+        and subsystems."
+      operationId: "compartmentInfo"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "id"
+        in: "path"
+        required: true
+        type: "string"
+      responses:
+        "200":
+          description: "Successful operation"
+        "404":
+          description: "Invalid input"
+  /compartments/{id}/related-reactions:
+    get:
+      tags:
+      - "compartment"
+      summary: "For a given compartment name (e.g., Golgi apparatus or
+        golgi_apparatus), return all reactions involving this compartment."
```

Comment:
I guess `golgi_apparatus` should be in quotes like`'Human-GEM'` above. The same applies for the rest of the queries.

Reply:
Seems not `Human-GEM`. The response for the query http://localhost/api/v2/compartments/golgi_apparatus
shows
```"compartmentSVGs": [
    {
      "customName": "Golgi apparatus",
      "filename": "golgi_apparatus.svg",
      "id": "golgi_apparatus"
    }
  ],
  ```

However, it seems the querying works only for the `id`, not `customName`.

Reply:
What I meant is that whatever the user is expected to type in should always be presented the same way, ie we should use the same quotes `'` everywhere.

>  it seems the querying works only for the `id`, not `customName`.

I think so too.